content/handbook/product/ux/user-journeys.md

```
@@ -0,0 +1,149 @@
+
+## ベストプラクティス
+
+最も効果的で有益なジャーニーマッピングは、仮定を超えて、実際のデータに基づいたユーザー体験マップを作成します。リサーチの深さと実用的なニーズのバランスを取り、実用的なインサイトを生み出します。
+
+### 始め方
+
+1. [ユーザージャーニー Issue テンプレート](https://gitlab.com/gitlab-org/gitlab/-/issues/new?description_template=User%20Journey)を使って Issue を作成することから始めます。
+    1. ユーザージャーニーの定義に誰が参加するかを決定。このチームと非同期または同期で作業して、ユーザージャーニーを完成させることができます。
+    1. ジョブ実行者、[JTBD](/handbook/product/ux/jobs-to-be-done/)、顧客セグメントなど、ジャーニーの基本を特定します。
+    1. リサーチ質問をまとめ、既存のデータを収集し、追加のリサーチが必要かどうかを判断します。
+    1. 提供されている [Figjam テンプレート](https://www.figma.com/board/CAw05ogtEWiRrhW48Uqt22/Journey-Mapping-Templates?t=EpYC7EjZOkOQkeUu-6)を使ってジャーニーをマッピング。ジャーニーマップテンプレートの使い方の手順については、以下の[ジャーニーマッピングの要点](#journey-mapping-essentials)セクションをレビューしてください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)